app/controllers/application_controller.rb

```diff
@@ -2,5 +2,9 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
-  before_filter :authenticate_user!
+  before_action :authenticate_user!
+
+  def authorize_admin
+    redirect_to root_path, alert: 'Доступ запрещен' unless current_user.admin?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.